Lib/gzip.py

```diff
@@ -17,6 +17,14 @@
 
 READ, WRITE = 1, 2
 
+# OS field values as specified in RFC 1952
+OS_CODES = {
+    'linux': 3,
+    'os2': 6,
```

Comment:
OS/2 no longer supported.

Reply:
I was looking at  RFC and used those code for which I was
certain that this is working.

Reply:
Fixed in 851c64b995b454e98d541236f48724bf2b59e3b2